ale/drivers/kaguya_driver.py

```diff
@@ -8,17 +8,29 @@
 from glob import glob
 
 from ale import config
-from ale.drivers.base import Driver, LineScanner, PDS3, Spice, TransverseDistortion
+from ale.drivers.base import Driver, LineScanner, Pds3Label, NaifSpice, TransverseDistortion
 
-class TcPds3Driver(Driver, LineScanner, PDS3, Spice, TransverseDistortion):
+class TcPds3Driver(Driver, LineScanner, Pds3Label, NaifSpice, TransverseDistortion):
     @property
     def instrument_id(self):
+        """
+        Id takes the form of LISM_<INSTRUMENT_ID>_<SD><COMPRESS><SWATH> where
+
+        INSTRUMENT_ID = TC1/TC2
+        SD = S/D short for single or double, which in turn means whether the
+             the label belongs to a mono or stereo image.
+        COMPRESS = D/T short for DCT or through, we assume labels belong
+        """
```

Comment:
Possibly, it would be true if we plan for the drivers themselves to be the documentation for the different gotchas on a particular mission or if we want a different place for that (I was going to mention it during the last stand-up and forgot, whoops). In either case, I don't think that should stop this from being merged.

Reply:
I agree, this is a nit picky thing. I like documentation